GDevelop.js/__tests__/GDJSLegacySceneVariableCodeGenerationIntegrationTests.js

```diff
@@ -0,0 +1,207 @@
+const initializeGDevelopJs = require('../../Binaries/embuild/GDevelop.js/libGD.js');
```

Comment:
This is the same exact old test file right?

Reply:
Yes, I renamed the other file, this way it's shown as a renaming operation.